transaction.js

```diff
@@ -37,11 +41,13 @@ class Transaction_AuroraDataMySQL extends Transaction { // eslint-disable-line c
         ...conn.parameters,
         transactionId: conn.transactions[conn.__knexTxId]
       };
+
```

Comment:
Minor feedback nit: whitespace additions shouldn’t be added in PRs unless they change semantics or behavior. But I’m ok with this stray instance in this PR, it’s minor and not worth fixing up.